sympy/physics/vector/point.py

```diff
@@ -22,30 +22,32 @@ class Point(object):
     Examples
     ========
 
-    >>> from sympy.physics.vector import Point, ReferenceFrame
+    >>> from sympy.physics.vector import Point, ReferenceFrame, dynamicsymbols
     >>> N = ReferenceFrame('N')
     >>> O = Point('O')
     >>> P = Point('P')
-    >>> O.set_vel(N, 10 * N.x + 20 * N.y + 30 * N.z)
-    >>> O.set_acc(N, 7 * N.x + 14 * N.y)
+    >>> u1, u2, u3 = dynamicsymbols('u1 u2 u3')
+    >>> v1, v2 = dynamicsymbols('v1 v2')
+    >>> O.set_vel(N, u1 * N.x + u2 * N.y + u3 * N.z)
+    >>> O.set_acc(N, v1 * N.x + v2 * N.y)
```

Comment:
I think it would be better to show `O.acc(N)` on this line. The accelerations will be automatically calculated.